### ISSUE TITLE
No license in the repo

### ISSUE THREAD

**Issue by k3d3** [eboody/styled#6]
Hello,

There doesn't appear to be any LICENSE file in this repo, and was hoping I could get some clarification on that.

I noticed that the Cargo.toml files mention the APL-1.0 license in it, which appears to be this: https://en.wikipedia.org/wiki/Adaptive_Public_License

Was that the intended license?

**Comment by eboody** [comment]
Yeah, so long as thats the most lax license available! Feel free to use this in any way you see fit! But for future reference I want it to be clear that thats the intention so I'll add license info to the repo as soon as possible
